static/settings.js

```diff
@@ -124,6 +126,40 @@ define(function (require) {
         });
         add(root.find('.hoverShowSource'), 'hoverShowSource', true, Checkbox);
         add(root.find('.hoverShowAsmDoc'), 'hoverShowAsmDoc', true, Checkbox);
+        var themeSelect = root.find('.theme');
+        add(themeSelect, 'theme', themes.default.id, Select,
+            _.map(themes, function (theme) {
+                return {label: theme.id, desc: theme.name};
+            })
+        );
+
+        function handleThemes() {
+            if (!colourSchemeSelect || !themeSelect) return;
```

Comment:
Yes! I tought that it's better to be safe than sorry, but if you think this is superflous, I can totally remove it 
❓